src/genai/_generated/endpoints.py

```diff
@@ -120,6 +120,36 @@ class RequestIdDeleteEndpoint(ApiEndpoint):
     version: str = "2023-11-22"
 
 
+class SystemPromptRetrieveEndpoint(ApiEndpoint):
```

Comment:
Should we add a service for these endpoints?

Reply:
I think that they should not be even part of the public API.
Ensure that this PR contains changes related to embeddings only.

Reply:
Ok, I removed other changes manually, we should try to keep the schemas up to date, but we can make another PR